pkg/render/dex_config.go

```diff
@@ -461,6 +480,19 @@ func (d *dexRelyingPartyConfig) UserInfoURI() string {
 	return fmt.Sprintf(userInfoURI, d.clusterDomain)
 }
 
+// CreateCertSecret creates the secret containing the certificate that others should mount in order to trust dex.
+func (d *dexConfig) CreateCertSecret() *corev1.Secret {
+	var certBytes []byte
+
+	if d.installation.CertificateManagement != nil {
```

Comment:
```suggestion
	if d.installation != nil && d.installation.CertificateManagement != nil {
```

Reply:
Installation can never be nil in any of the render package. All controllers will terminate if it  is not present.